src/go/k8s/apis/redpanda/v1alpha1/cluster_webhook.go

```diff
@@ -105,6 +106,13 @@ func (r *Cluster) Default() {
 			Resources: DefaultRpkStatusResources,
 		}
 	}
+	if r.Spec.PodDisruptionBudget == nil {
+		defaultMaxUnavailable := intstr.FromInt(1)
+		r.Spec.PodDisruptionBudget = &PDBConfig{
+			Enabled:        true,
+			MaxUnavailable: &defaultMaxUnavailable,
+		}
+	}
```

Comment:
NIT: should we add some sort of warning if MaxUnavailable or MinAvailable is set but Enabled is set to false?

Reply:
I've made it an error